R/SS_read.R

```diff
@@ -43,6 +42,18 @@
 #' inputs <- SS_read(
 #'   dir = system.file("extdata", "simple_3.30.13", package = "r4ss")
 #' )
+#' # Read in an example from GitHub stored in user-examples,
+#' # wrapped in `dontrun` because it requires an Internet connection
```

Comment:
Is this a best practice if something requires an internet connection?

Reply:
imo yes; but, I am not sure if this is a documented best practice, just what I do because sometimes I am testing packages without an internet connection.